.github/workflows/test-cloud-platforms.yml

```diff
@@ -0,0 +1,149 @@
+name: Test cloud platforms
+env:
+  REGISTRY: ghcr.io
+  CONTAINER_REPO: ${{ github.repository }}
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+on:
+  schedule:
+    # Each week day at 9:30 UTC (so 3:30 MDT and 2:30 MST).
+    - cron: "30 9 * * MON-FRI"
+
+# Jobs are given a level in a comment.
+# Jobs of the same level run in parallel.
+# Jobs of level N depend of, at least, one job on level N - 1 expect job whom
+# level is 0.
+jobs:
+  check-secrets:
+    name: Check repo secrets
+    # level: 0
+    runs-on: ubuntu-latest
+    outputs:
+      aks: ${{ steps.set_output.outputs.aks }}
+    steps:
+      # Secrets cannot be used as if condition, use job output as workaround.
+      # https://github.com/actions/runner/issues/520
+      - id: set_output
+        run: |
+          if [[ "${{ secrets.AZURE_AKS_CREDS }}" != "" && \
+                "${{ secrets.AZURE_AKS_RESOURCE_GROUP }}" != "" && \
+                "${{ secrets.AZURE_AKS_CLUSTER_PREFIX }}" != "" ]]; \
+          then
+            echo "Secrets to use an ARO cluster were configured in the repo"
+            echo "::set-output name=aks::true"
+          else
+            echo "Secrets to use an ARO cluster were not configured in the repo"
+            echo "::set-output name=aks::false"
+          fi
+
+  build-kubectl-gadget:
```

Comment:
Is it possible to create a specific action for this that receives the container image and the tag to be used? Is it possible to upload the binary to the artifact from there?

For this workflow, it will pass `CONTAINER_REPO=${{ env.REGISTRY }}/${{ env.CONTAINER_REPO }}` and `IMAGE_TAG=latest`. While for the _Inspektor Gadget CI_ workflow, they would be the output of the `set-repo-determine-image-tag` step.